src/definitions.jl

```diff
@@ -419,8 +419,34 @@ Base.copy(x::Frequencies) = x
     fftfreq(n, fs=1)
 Return the discrete Fourier transform (DFT) sample frequencies for a DFT of length `n`. The returned
 `Frequencies` object is an `AbstractVector` containing the frequency
-bin centers at every sample point. `fs` is the sample rate of the
-input signal.
+bin centers at every sample point. `fs` is the sampling rate of the
+input signal, which is the reciprocal of the sample spacing.
+
+Given a window of length `n` and a sampling rate `fs`, the frequencies returned are 
+
+```julia
+[0:n÷2-1; -n÷2:-1]  .* fs/n   # if n is even
```

Comment:
No need for `.*`

Reply:
that is, `*` also works here and is simpler

Reply:
Ah indeed, I've updated it now. Thanks again!